batch/test/test_database.py

```diff
@@ -0,0 +1,124 @@
+import os
+import unittest
+
+from batch.database import Database, run_synchronous
+
+
+class Test(unittest.TestCase):
+    def setUp(self):
+        self.db = Database.create_synchronous(os.environ.get('CLOUD_SQL_CONFIG_PATH'))
```

Comment:
I deleted this file because we don't have a way to make new tables with the permissions on the current batch setup